pkg/agent/imagebuilder/releaseimage_test.go

```diff
@@ -50,7 +56,6 @@ func TestReleaseImageListErrors(t *testing.T) {
 	cases := []string{
 		"",
 		"quay.io/openshift-release-dev/ocp-release-4.10",
-		"quay.io/openshift-release-dev/ocp-release:4",
```

Comment:
If we check the length of the digest we wouldn't need to remove this test.

Reply:
Thanks, and I also noticed I forgot to remove the skip in the previous test!